tests/context-menu.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+import { waitForCustomEvent } from './lib/helper.js';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/playground/context-menu-test.html');
+});
+
+test('Renders labels, dividers, and icons', async ({ page }) => {
+  const contextMenu = page.getByTestId('context-menu-default');
+  await expect(contextMenu).toBeVisible();
+  await expect(contextMenu.getByText('label 1')).toBeVisible();
+  await expect(contextMenu.getByTestId('context-menu-divider')).toBeVisible();
```

Comment:
An `hr` should have an implicit role of `separator`, so we should use that instead of `test-id`